### PR TITLE
V kalc/app in meeting

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**
App in meeting.

### CODE REVIEW

File: samples/app-in-meeting/csharp/AppInMeeting/AppPackage/manifest.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "full": "App In Meeting"
+  },
+  "description": {
+    "short": "App In Meeting",
+    "full": "This example shows app to add before, after and during a meeting"
+  },
+  "accentColor": "#5558AF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://cf40b0aa1718.ngrok.io/configure",
```
Comment (v-chetsh) [inline samples/app-in-meeting/csharp/AppInMeeting/AppPackage/manifest.json:28]: We can remove the ngrok id here & add placeholder.

File: samples/app-in-meeting/csharp/AppInMeeting/Pages/AppInMeeting.cshtml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    });
+                }
+                else {
+                    $(this).css({
+                        "border": "",
+                        "background": ""
+                    });
+                }
+            });
+            if (isValid == false) {
```
Comment (v-chetsh) [inline samples/app-in-meeting/csharp/AppInMeeting/Pages/AppInMeeting.cshtml:107]: One line space before it.

**Review by v-chetsh — CHANGES_REQUESTED** [review]
Provided few comments, please fix those.

File: README.md [hunk 3]
```diff
@@ -188,6 +189,7 @@
 [meetings-sidepanel#cs]:samples/meetings-sidepanel/csharp
 [meetings-sidepanel#js]:samples/meetings-sidepanel/nodejs
 [meetings-content-bubble#js]:samples/meetings-content-bubble/nodejs
+[app-in-meeting#cs]:samples/app-in-meeting/csharp
```
Comment (v-chetsh) [inline README.md:192]: Looks like we haven't added link for nodejs sample for app-in-meeting, please check & update if not added.

**Review by v-chetsh — APPROVED** [review]
Approving the PR with minor change.

File: samples/app-in-meeting/csharp/AppInMeeting/AppPackage/manifest.json (hunk 1 above)
Comment (AbhijeetThakur-msft) [reply samples/app-in-meeting/csharp/AppInMeeting/AppPackage/manifest.json]: done

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**